v2/test/README.md

```diff
@@ -54,6 +63,12 @@ Run sample test by:
 make sample-test
 ```
 
+Run integration test by (however, integration tests are configured to run on kfp-ci project, so modify samples locally with your own configs):
```

Comment:
Maybe we can add a link to the config that needs to be modified? Or a link to section below in this README file?

Reply:
Good point! Added